lib/workers/pr/body/controls.ts

```diff
@@ -1,4 +1,4 @@
-import { emojify } from 'node-emoji';
+import { emojify } from '../../../util/emoji';
```

Comment:
It potentially changes `getControls` output (hopefully, towards more correct emoji handling)

Reply:
So it will only update pr description, which is probably ok. @rarkins What do you think?